### PR TITLE
hotfix:400-error

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Closes #https://github.com/influxdata/DAR/issues/312, https://github.com/influxdata/openapi/pull/447

- Hotfix Kelly and I worked on to update 400 partial write error description in the API docs.
- Updates to Scripts API are also included since I pulled all changes instead of just cherry picking the single hotfix commit.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: api-docs/cloud/ref.yml [hunk 1]
```diff
@@ -4144,13 +4144,12 @@ components:
     ScriptCreateRequest:
       properties:
         description:
+          description: Script description. A description of the script.
```
Comment (sanderson) [inline api-docs/cloud/ref.yml:4147]: Seems redundant

```suggestion
          description: Script description.
```

File: api-docs/cloud/ref.yml [hunk 2]
```diff
@@ -11039,35 +11038,135 @@ paths:
         - System information endpoints
   /api/v2/scripts:
     get:
+      description: >
+        Retrieves a list of
+        [scripts](/influxdb/cloud/api-guide/api-invokable-scripts/).
```
Comment (sanderson) [inline api-docs/cloud/ref.yml:11043]:
```suggestion
        [invokable scripts](/influxdb/cloud/api-guide/api-invokable-scripts/).
```

File: api-docs/cloud/ref.yml [hunk 3]
```diff
@@ -11039,35 +11038,135 @@ paths:
...
+
+
+        #### Limitations
+
+
+        - Paging with an `offset` greater than the number of records will result
+        in
+
+        an empty response--for example:
```
Comment (sanderson) [inline api-docs/cloud/ref.yml:11052]: Is the extra whitespace here on purpose?

File: api-docs/cloud/ref.yml [hunk 4]
```diff
@@ -14635,9 +14810,14 @@ tags:
...
-      missing or malformed or the API token does not have permission for the
-      operation. |
+      | `400`       | Bad request              | May indicate one of the
+      following: <li>Line protocol is malformed. The response body contains the
+      first malformed line in the data and indicates what was expected. For
+      partial writes, the number of points written and the number of points
+      rejected are also included. For more information, check the
+      `rejected_points` measurement in your `_monitoring`
+      bucket.</li><li>`Authorization` header is missing or malformed or the API
+      token does not have permission for the operation.</li> |
```
Comment (sanderson) [inline api-docs/cloud/ref.yml:14820]: I know redoc will take care of it, but I think we should make the inline HTML valid.

```suggestion
      | `400`       | Bad request              | May indicate one of the
      following: <ul><li>Line protocol is malformed. The response body contains the
      first malformed line in the data and indicates what was expected. For
      partial writes, the number of points written and the number of points
      rejected are also included. For more information, check the
      `rejected_points` measurement in your `_monitoring`
      bucket.</li><li>`Authorization` header is missing or malformed or the API
      token does not have permission for the operation.</li></ul> |
```

File: api-docs/v2.3/ref.yml [hunk 5]
```diff
@@ -16494,9 +16499,14 @@ tags:
...
-      missing or malformed or the API token does not have permission for the
-      operation. |
+      | `400`       | Bad request              | May indicate one of the
+      following: <li>Line protocol is malformed. The response body contains the
+      first malformed line in the data and indicates what was expected. For
+      partial writes, the number of points written and the number of points
+      rejected are also included. For more information, check the
+      `rejected_points` measurement in your `_monitoring`
+      bucket.</li><li>`Authorization` header is missing or malformed or the API
+      token does not have permission for the operation.</li> |
```
Comment (sanderson) [inline api-docs/v2.3/ref.yml:16509]: Same suggestion as above.

**Review by sanderson — APPROVED** [review]
Some minor suggestions. I know these were auto-gen'd, so take them with a grain of salt :)

File: api-docs/cloud/ref.yml (hunk 2 above)
Comment (jstirnaman) [reply api-docs/cloud/ref.yml:11043]: Let's keep `scripts` as they're likely to decouple from the invokable-ness.

File: api-docs/cloud/ref.yml (hunk 3 above)
Comment (jstirnaman) [reply api-docs/cloud/ref.yml:11052]: I think 

> Is the extra whitespace here on purpose?

I think it gets added by openapi-cli.

File: api-docs/cloud/ref.yml (hunk 4 above)
Comment (jstirnaman) [reply api-docs/cloud/ref.yml:14820]: > I know redoc will take care of it, but I think we should make the inline HTML valid.

I'll fix this in my next openapi PR

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**